### DISCUSSION TITLE
Not able to run Foocus

### DISCUSSION THREAD

**Discussion by SkCodeRains** [lllyasviel/Fooocus#2934] [Q&A] [answered]
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

i am having i3-12100 CPU and 1TB 980 PRO nvme No GPU , CPU inbuilt having some graphics .
i ran run.batch and getting this error !! 

So is it necessary to have nvidia or any other graphics !! ?? i want to use it without graphics card , it is okey to take time while processing !! i am not in hurry !!

**Comment by Rogala** [comment, marked as answer]
[Edit run.bat and add: --always-cpu](https://github.com/lllyasviel/Fooocus/discussions/2554)
